pkg/lib/_internal/common.ts

```diff
@@ -26,27 +28,27 @@ export function in_array(array, val) {
     return false;
 }
 
-export function is_function(x) {
+export function is_function(x: unknown): x is (...args: unknown[]) => unknown {
     return typeof x === 'function';
 }
 
-export function is_object(x) {
+export function is_object(x: unknown): x is object {
     return x !== null && typeof x === 'object';
 }
 
-export function is_plain_object(x) {
+export function is_plain_object(x: unknown): boolean {
```

Comment:
The "plain boolean" return type is a bit inconsistent with is_object(), but then again, who knows what this function actually does.. :grin: :see_no_evil:

Reply:
I chose this return type because I couldn't come up with a better type.  It's hard to describe "an object, but not the fancy kind" :rofl:

Reply:
Yeah, I don't even know what it does in plain English :grin: